modules/highlight-mfa-users/highlight-mfa-users.php

```diff
@@ -0,0 +1,131 @@
+<?php
+namespace Automattic\VIP\Security\MFAUsers;
+
+class Highlight_MFA_Users {
+	private static $mode;
+	const MFA_SKIP_USER_IDS_OPTION_KEY = 'vip_security_mfa_skip_user_ids';
+	public static function init() {
+		if ( ! defined( 'VIP_SECURITY_BOOST_CONFIGS' ) ) {
+			error_log( 'VIP_SECURITY_BOOST_CONFIGS not defined' );
+			return;
+		}
+
+		$configs             = constant( 'VIP_SECURITY_BOOST_CONFIGS' );
+		$highlight_mfa_users = $configs['highlight_mfa_users'] ?? [];
+		self::$mode          = $highlight_mfa_users['mode'] ?? 'REPORT';
```

Comment:
I'm wondering if we need a `mode` config here. Do we foresee other modes we'll be adding in the future?